app/helpers/physical_storage_consumer_helper/textual_summary.rb

```diff
@@ -0,0 +1,58 @@
+module PhysicalStorageConsumerHelper::TextualSummary
+  include TextualMixins::TextualGroupTags
+
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name
+        ems
+        physical_storage
+      ]
+    )
+  end
+
+  def textual_group_relationships
+  end
+
+  def textual_group_addresses
+    addresses_values = []
+
+    addresses = Address.where(physical_storage_consumer_id: @record.id)
+    addresses.each do |address|
+      if defined?(address.port.iqn) && address.port.iqn
+        addresses_values << [_("iqn"), address.port.iqn]
+      elsif defined?(address.port.wwpn) && address.port.wwpn
+        addresses_values << [_("wwpn"), address.port.wwpn]
+      end
+    end
```

Comment:
This will need to be adjusted when you add a uniform method for getting the address.

Reply:
done